src/Admin/OptionsMenu.php

```diff
@@ -321,9 +328,9 @@ public function render_screen() {
 											<?php esc_html_e( 'Open Wizard', 'amp' ); ?>
 
 										<?php endif; ?>
-										<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64">
-											<path d="M43.16 10.18c-0.881-0.881-2.322-0.881-3.203 0s-0.881 2.322 0 3.203l16.335 16.335h-54.051c-1.281 0-2.242 1.041-2.242 2.242 0 1.281 0.961 2.322 2.242 2.322h54.051l-16.415 16.335c-0.881 0.881-0.881 2.322 0 3.203s2.322 0.881 3.203 0l20.259-20.259c0.881-0.881 0.881-2.322 0-3.203l-20.179-20.179z" />
-										</svg>
+									</a>
+									<a class="components-button is-link" href="<?php echo esc_url( amp_get_customizer_url() ); ?>" target="_blank">
```

Comment:
I don't think this is quite right. Given that the Customizer URL varies by the mode, I think the Customizer URL needs to be returned as a `readonly` property from the `options` endpoint in the same way that the `preview_permalink` is being returned. Then this link can be conditionally showed based on whether the the Options endpoint returns this URL as being non-empty and it can be updated when the template mode and reader theme are changed.

Reply:
Oh, then I see that will depend on moving this whole `settings-welcome` into being part of the React app as well.

Reply:
Done in f0d318b52